rl_baselines/base_classes.py

```diff
@@ -235,7 +243,10 @@ def train(self, args, callback, env_kwargs=None, train_kwargs=None):
                      'mlp': MlpPolicy,
                      'lstm': MlpLstmPolicy,
                      'lnlstm': MlpLnLstmPolicy}[args.policy]
-
-        self.model = self.model_class(policy_fn, envs, **train_kwargs)
+        if self.load_rl_model_path is not None:
+            print("Load trained model from the path: ", self.load_rl_model_path)
+            self.model = self.model_class.load(self.load_rl_model_path, envs, **train_kwargs)
+        else:
+            self.model = self.model_class(policy_fn, envs, **train_kwargs)
```

Comment:
please use strings for the policy type, otherwise, this may break for algos like DQN.

Reply:
Are you referring to the values of this dict 
```policy_fn = {'cnn': CnnPolicy,
                     'cnn-lstm': CnnLstmPolicy,
                     'cnn-lnlstm': CnnLnLstmPolicy,
                     'mlp': MlpPolicy,
                     'lstm': MlpLstmPolicy,
                     'lnlstm': MlpLnLstmPolicy}```
to be set as strings ?

Reply:
yep, stable-baselines supports strings now, and it avoid importing different policies for different models